source/nodejs/spec-generator/generate.cmd

```diff
@@ -0,0 +1,16 @@
+IF EXIST "dist\spec-generator-script.0.5.0.0.min.js" (
```

Comment:
So I'm wondering if there's any way to have node run this logic? If we write this as a .cmd then our Mac and Linux friends will be unable to generate specs. npm has a way of declaring "bin" functions which are designed to be executed like command line scripts, except x-plat. Take a look here for an example: 

https://github.com/matthidinger/marked-schema/blob/master/bin/marked-schema

Reply:
That's a great future improvement, but we need to make progress now. That marked schema example doesn't **build** the node js project. That's the key commands in the current script. The current script checks if a pre-built version of the script exists and if not builds it on-demand.

I'm sure it's possible to make that work, but we don't have time right now (or I don't).